README.md

```diff
@@ -1,6 +1,6 @@
-<div align="center">
+<div style="display: grid; place-items: center;">
```

Comment:
Did you intend to include these changes in this PR?

Reply:
Done: https://github.com/hyperlight-dev/hyperlight/pull/36